app/assets/stylesheets/base/_reset.scss

```diff
@@ -9,3 +8,8 @@ body {
   flex-direction: column;
   justify-content: space-between;
 }
+
+
+h1, h2, h3, h4, h5, h6 {
+  margin: 0;
+}
```

Comment:
Add an empty line at the end of your code

Reply:
done